### ISSUE TITLE
The auto-daily can't be set up on new channels/guilds.

### ISSUE THREAD

**Issue by Zukunftsmusik** [PieInTheSky-Inc/YaDc#86]
Error message:
```
Error: Command raised an exception: AttributeError: 'tuple' object has no attribute 'created_at'
```

**Comment by Zukunftsmusik** [comment]
Fixed for v1.2.6.1

**Comment by Zukunftsmusik** [comment]
Reopened with another error message:
```
Error: Command raised an exception: AttributeError: 'NoneType' object has no attribute 'day'
```

**Comment by Zukunftsmusik** [comment]
Fixed for v1.2.7.4
